views/partials/_contact-panel.njk

```diff
@@ -9,7 +9,7 @@
     <ul class="govuk-list govuk-list--bullet">
       <li>
         <a class="govuk-link app-contact-panel__link" href="https://github.com/alphagov/govuk-design-system-backlog/issues/{{ backlog_issue_id }}">
-          share your research or feedback on GitHub
+          take part in the {{ title }} discussion on GitHub and share your research
```

Comment:
Thanks Eoin!

I spot-checked a few pages in the preview to get an idea about how {{ title }} would look, and it'd be a good idea to add the single quotes as suggested before.

```suggestion
          take part in the {{ title }} discussion on GitHub and share your research
```